### ISSUE TITLE
Can I use 256KB or 512KB menu rom?

### ISSUE THREAD

**Issue by Yhc-Studio** [ClusterM/coolboy-multirom-builder#11]
I wonder if this compiler can use 256KB or 512KB Rom?
How can I modify it if I can?

**Comment by ClusterM** [comment]
Hi!
Which ROM you mean? Game ROM? Result ROM? Menu ROM?

**Comment by Yhc-Studio** [comment]
> Hi! Which ROM you mean? Game ROM? Result ROM? Menu ROM?
Yes.Menu ROM. And when I used the CoolboyCombiner tool, I found that menus larger than 128KB could not be merged. I want to know how to modify?

**Comment by ClusterM** [comment]
In theory, you can create menu ROM of unlimited size. But this constructor uses fixes menu size. I can't understand, what the point?

**Comment by Yhc-Studio** [comment]
I understand. I also know this Mapper, the maximum menu can only support 512KB. As for doing 256KB or even 512KB menu ROMs, it's more about trying to cram in larger NSF music data, I'm just asking how can I use 256KB and 512KB menu ROMs to merge the resulting ROMs?

**Comment by Yhc-Studio** [comment]
To do that, you'd have to change the program source itself, right?
Because I do want to merge the final result ROM with the menu ROM I made myself.

**Comment by ClusterM** [comment]
> To do that, you'd have to change the program source itself, right?
> Because I do want to merge the final result ROM with the menu ROM I made myself.

Yes

**Comment by Yhc-Studio** [comment]
Thanks, I have known how to change loader rom size.
